.github/workflows/label.yml

```diff
@@ -27,4 +27,3 @@ jobs:
     - uses: actions/labeler@v5
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        sync-labels: true
```

Comment:
Should we keep the field, but set it to false, so it remains obvious what is happening and easier to know what to do if we reverse course for some reason in the future?